index.html

```diff
@@ -619,30 +619,41 @@ <h4>Processing considerations</h4>
   </section>
   <section>
     <h2>Exposing change of MediaStreamTrack configuration</h2>
+    <p>The configuration (capabilities, constraints or settings) of a {{MediaStreamTrack}} may be changed dynamically
+      outside the control of web applications. One example is when a user decides to switch on background blur through
+      the operating system. Web applications might want to know that the configuration
+      of a particular {{MediaStreamTrack}} has changed. For that purpose, a new event is defined below.
+   </p>
+    <h3>MediaStreamTrack Interface Extensions</h3>
     <div>
-      <p>The configuration (capabilities, constraints or settings) of a {{MediaStreamTrack}} may be changed dynamically
-        outside the control of web applications. One example is when a user decides to switch on background blur through
-        the operating system. Web applications might want to know that the configuration
-        of a particular {{MediaStreamTrack}} has changed. For that purpose, a new event is defined below.
-     </p>
-      <pre class="idl">
+    <pre class="idl">
 partial interface MediaStreamTrack {
   attribute EventHandler onconfigurationchange;
 };</pre>
+    <p>The <dfn data-dfn-for="MediaStreamTrack">onconfigurationchange</dfn> attribute
+      is an [=event handler IDL attribute=] for the `onconfigurationchange`
+      [=event handler=], whose [=event handler event type=] is
+      <dfn event for=MediaStreamTrack>configurationchange</dfn>.
+    </p>
     <p>
       <p>When the [=User Agent=] detects <dfn data-export id="change-track-configuration">a change of configuration</dfn>
        in a <var>track</var>'s underlying source, the [=User Agent=] MUST run the following steps:</p>
```

Comment:
This is a pre-existing issue but it is not very clear whether each clone track will fire in its own event loop task or in the same event loop task.
We might want to file an issue about this.